### ISSUE TITLE
Monitoring several accounts

### ISSUE THREAD

**Issue by tecaspt** [fernandod1/Instagram-to-discord#23]
Is there any particular way to monitor several accounts? I managed to do so by running a few instances, but I was wondering if there was a way to run on only one, or a way to more easily deploy several instances more easily

**Comment by aternis** [comment]
Yes. It's possible to do this. Just not at the current state of the app

**Comment by xhendrikg** [comment]
> Is there any particular way to monitor several accounts? I managed to do so by running a few instances, but I was wondering if there was a way to run on only one, or a way to more easily deploy several instances more easily

Just Execute it several times?

**Comment by timpiglowski** [comment]
You can use `screen` (`apt install screen`) to execute the code simultaneously.
